### ISSUE TITLE
Include instructions on how to setup latest chdman with game-convert

### ISSUE THREAD

**Issue by Codythedragonrude** [Justme488/game-converter#8]
Install mame-tools whatever the latest is available for your distro 

Download game-convert source folder

Extract it to wherever (i prefer home directory since it's easiest to manage)

Now navigate to your bin folder in your root directory (this is where mame-tools puts chdman for pretty much all the distros i tested this with)

copy chdman from bin to the game-convert source folder rename it to chdman version latest if you want to keep the other two chdman options or just delete chdman 4 and 5 if you only want to use latest chdman

Before we do anything else the only files you need in game-converter source folder are 
chdman version latest game-converter.sh and game-converter.png(optional if you want to make an easy to find menu shortcut in desktop environments like XFCE) 

Now open the game-convert.sh into whatever text editor you use in my case i just use mousepad 

in this sh file you should see the following entry 
# Define chdman version variables
opt_chdman4="chdman version 4 (0.145)"
opt_chdman5="chdman version 5 (0.221)"

We need to point these to the location of the chdman files
So basically in my case it would go from

# Define chdman version variables
opt_chdman4="chdman version 4 (0.145)"
opt_chdman5="chdman version 5 (0.221)"

To 

opt_chdman="/home/usernamehere/game-converter-latest/chdman version 4 (0.145)"
opt_chdman="/home/usernamehere/game-converter-latest/chdman version 5 (0.221)"
opt_chdman="/home/usernamehere/game-converter-latest/chdman version latest (Latest)"

Then save that 

Finally test from terminal by running 
sh game-converter-sh

The end result should be the zenity gui but this time with a third option in the chdman selection menu called chdman latest

**Comment by Codythedragonrude** [comment]
So that seems to break the chdman selection screen i may need to mess with some more sh script stuff then

**Comment by Codythedragonrude** [comment]
seems like this part might be the culprit ` # If chdman5 was selected, start creating chd
      elif [[ "$chdman_ver" == "$opt_chdman5" ]]; then
        (chdman5 createcd -i "$single_tochd_input" -o "$final_single_tochd_output"| zenity --progress --pulsate --auto-kill --width="400" --auto-close --title="Converting $single_tochd_input_basename_no_ext" --text="Creating: $single_tochd_input_basename_no_ext.chd")
        # Cancel conversion, and delete incomplete file if cancel is pressed.
        if [[ "$?" != 0 ]]; then
          rm "$final_single_tochd_output"
          pkill chdman5
        fi
      fi`

need to probably remove those two entries and point them to the proper path 

i'm probably going to do a fully written guide about this then
